src/cond.d

```diff
@@ -25,6 +25,20 @@ import ddmd.root.outbuffer;
 import ddmd.tokens;
 import ddmd.visitor;
 
+private __gshared Identifier idUnitTest;
```

Comment:
But that's exactly what the gigantic list of identifiers in `Id` are for.  What is the point of not using that?